test/extended/util/test.go

```diff
@@ -384,25 +384,10 @@ var (
 
 			`Services should be able to create a functioning NodePort service`, // https://github.com/openshift/origin/issues/21708
 
-			`SSH`,                // TRIAGE
-			`SELinux relabeling`, // https://github.com/openshift/origin/issues/7287 still broken
-			`Volumes CephFS`,     // permission denied, selinux?
-
-			`should support inline execution and attach`, // https://bugzilla.redhat.com/show_bug.cgi?id=1624041
+			`SSH`,            // TRIAGE
+			`Volumes CephFS`, // permission denied, selinux?
 
 			`should idle the service and DeploymentConfig properly`, // idling with a single service and DeploymentConfig [Conformance]
-
-			`\[Feature:Volumes\]`, // storage team to investigate it post-rebase
-
-			// TODO: the following list of tests is disabled temporarily due to the fact
-			// that we're running kubelet 1.11 and these require 1.12. We will remove them
-			// post-rebase
-			`\[Feature:NodeAuthenticator\]`,
-			`PreemptionExecutionPath`,
-			`\[Volume type: blockfswithoutformat\]`,
-			`CSI Volumes CSI attach test using HostPath driver`,
-			`CSI Volumes CSI plugin test using CSI driver: hostPath`,
-			`Volume metrics should create volume metrics in Volume Manager`,
```

Comment:
I'm trying to re-enable those tests in https://github.com/openshift/origin/pull/21904 but they continue to fail. I've pinged  people but haven't heard back.